client/my-sites/checkout/composite-checkout/components/wp-order-review-line-items.tsx

```diff
@@ -154,18 +154,39 @@ WPOrderReviewLineItems.propTypes = {
  * @param cartProducts The shopping cart object
  * @returns boolean
  */
-function isPremiumPlanWithDIFMInTheCart( item: ResponseCartProduct, cartProducts: ResponseCart ) {
```

Comment:
This is throwing a linting error because the name of the second param changed and it no longer matches the JSDoc. Since this is TypeScript now, let's remove the JSDoc param/returns lines completely.